src/Nethermind/Nethermind.Evm.Test/TransactionProcessorEip7623Tests.cs

```diff
@@ -63,7 +63,7 @@ public void transaction_validation_intrinsic_below_floor(long gasLimit, bool isF
             .WithGasLimit(10000000).TestObject;
 
         TransactionResult result = _transactionProcessor.Execute(tx, new BlockExecutionContext(block.Header, _specProvider.GetSpec(block.Header)), NullTxTracer.Instance);
-        Assert.That(result.Fail, Is.EqualTo(isFail));
+        Assert.That(result.TransactionExecuted, Is.EqualTo(!isFail));
```

Comment:
Nitpick: invert the `isFail` in the test cases itself to `executed` (or similar) so that the assertion looks like:


```suggestion
        Assert.That(result.TransactionExecuted, Is.EqualTo(executed));
```

Reply:
done